libraries/lib-datahandler/src/main/thrift/components.thrift

```diff
@@ -194,6 +196,8 @@ struct Release {
     54: optional COTSDetails cotsDetails,
 
     65: optional set<string> mainLicenseIds,
+    66: optional set<string> mainLicenseNames,
```

Comment:
I thought I've successfully deleted this field in #135. Why are you adding it back?